### PR TITLE
Fix heading spelling of keypadc.h docs

### PR DESCRIPTION
**Pull request by myclevorname**
Dection → Detection

Pretty self-explanatory.

### CODE REVIEW

File: docs/libraries/keypadc.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ It supports multi-key presses, and can be used for extremely responsive input co
 .. contents:: :local:
    :depth: 3
 
-Helpful key dection
+Helpful key detection
```
Comment (runer112) [inline docs/libraries/keypadc.rst:16]: I just noticed, why is the word "Helpful" here? Is it just me thinking that it sounds weird with it?

Comment (myclevorname) [reply docs/libraries/keypadc.rst:16]: I will change that too.

**Review by runer112 — APPROVED** [review]

**Comment by myclevorname** [comment]
Maybe it could be 'Detecting Keys' instead of 'Key Detection'. What are your thoughts?

**Comment by runer112** [comment]
> Maybe it could be 'Detecting Keys' instead of 'Key Detection'. What are your thoughts?

I think what you already have with "Key Detection" is good.

**Comment by runer112** [comment]
I'll wait a bit for any other potential reviewers to get a chance, but short of any other reviews, I'm happy to merge this. Seems like a simple and objective improvement.

**Pull request merged**